src/DefaultEventLogger.ts

```diff
@@ -54,6 +55,37 @@ class DefaultEventLogger implements EventLogger, EventPreProcessor, EventPostPro
       return result
     }
 
+    createNewTraceMetadata(service: string, sampled?: number | undefined, flags?: number | undefined, timestamp?: string | undefined): EventTraceMetadata {
+
+      let newMeta = new EventTraceMetadata(service, new Uuid(), new Uuid(), undefined, sampled, flags, timestamp);
```

Comment:
I believe we will not be able to utilise UUIDs for any of the trace/span information and have to adopt the following method: https://github.com/elastic/node-traceparent#example-usage

`crypto.randomBytes(16).toString('hex')` for either traceId, spanId, or parentSpanId going forward to align to the wc3 standard for tracing.

Reply:
I agree

Reply:
DONE

Reply:
`6.4.6-snapshot`